src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -359,9 +359,21 @@ public OpenIdConnectHandler(HttpClient backchannel, IHtmlEncoder htmlEncoder)
                 // if any of the error fields are set, throw error null
                 if (!string.IsNullOrEmpty(message.Error))
                 {
-                    // REVIEW: this error formatting is pretty nuts
-                    Logger.LogError(13, "Message contains error: '{0}', error_description: '{1}', error_uri: '{2}'.", message.Error, message.ErrorDescription ?? "ErrorDecription null", message.ErrorUri ?? "ErrorUri null");
-                    return AuthenticateResult.Failed(new OpenIdConnectProtocolException(string.Format(CultureInfo.InvariantCulture, Resources.MessageContainsError, message.Error, message.ErrorDescription ?? "ErrorDecription null", message.ErrorUri ?? "ErrorUri null")));
+                    Logger.LogError(
+                        13,
```

Comment:
FWIW, why not removing the (horrible) `"ErrorDecription null"` and `"ErrorUri null"` default values? IMHO, they don't make much sense in a "semantic log"... and it would help reducing the length of this line :smile: